packages/components/select/src/useSelect.ts

```diff
@@ -811,30 +809,24 @@ export const useSelect = (props, states: States, ctx) => {
       .filter((option) => option.visible)
       .every((option) => option.disabled)
   )
-
-  const navigateOptions = (direction) => {
```

Comment:
This condition of `!states.visible` was merged by me into the following if condition. Do you think it would work?